source/Distribution.h

```diff
@@ -0,0 +1,41 @@
+/* Distribution.h
+Copyright (c) 2022 by DeBlister
+
+Endless Sky is free software: you can redistribute it and/or modify it under the
+terms of the GNU General Public License as published by the Free Software
+Foundation, either version 3 of the License, or (at your option) any later version.
+
+Endless Sky is distributed in the hope that it will be useful, but WITHOUT ANY
+WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A
+PARTICULAR PURPOSE.  See the GNU General Public License for more details.
+*/
+
+#ifndef DISTRIBUTION_H_
+#define DISTRIBUTION_H_
+
+#include "Angle.h"
+
+#include <tuple>
```

Comment:
```suggestion
#include <utility>
```

`std::pair` is in utility, not in tuple.

Reply:
I see the thumbs up but I would appreciate an explanation of the reasoning why <tuple> can't be used as the original comment is not fully correct in my understanding.  If that's not how it works then please let me know.